mobx_codegen/test/mobx_codegen_test.dart

```diff
@@ -37,7 +48,15 @@ void main() {
       const TestInfo(
           description: 'generates for a generic class mixing Store',
           source: './data/valid_generic_store_input.dart',
-          output: './data/valid_generic_store_output.dart')
+          output: './data/valid_generic_store_output.dart'),
+      const TestInfo(
```

Comment:
Curious, if you find this way of creating tests good? Are there better approaches?

Reply:
I think this approach is great. Easy to add, and to change over time. I only wish the diff were a little better in the failure case.

Reply:
Ya the failure case is not ideal, only because its a bunch of errors and not in a fixed order. Perhaps we should sort the ordering of error messages in the output 🤔. That should make it easier to create failure output files easily ?

Reply:
I actually spent some time debugging one of these test failures. I had to get expected and actual side-by-side to notice the very slight difference I needed to satisfy. I thought to myself, huh, wish it showed me the first difference.

Also, it should be easy to copy the actual string out of the terminal. Right now it's rendered as a string concat in this style:

'this is a line\n'
'this is another line\n'
It'd be nice if it were a multiline, triple-quote literal. Or no quotes at all. That way it's easier to copy and paste.

Reply:
Sorry, brain dump on the test format. Fact is, I do like it. It's a useful way to define tests.

Reply:
Ya I have been through that 😄, so I know **exactly** what you underwent!